src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter((person) => person.sex === 'm');
+
+  if (century) {
+    men = men.filter((person) => Math.ceil(person.died / 100) === century);
+  }
+
+  const sumAge = men.reduce((sum, man) => sum + (man.died - man.born), 0);
```

Comment:
You can destructure the `man` object in the callback arguments to have only `died` and `born` variables.